packages/styles/components/_c.tooltip.scss

```diff
@@ -70,6 +82,12 @@
         left: 50%;
         transform: translate(-50%, 100%) rotate(-90deg);
       }
+
+      &::before {
+        bottom: -1px;
+        left: 50%;
+        transform: translate(-50%, 100%) rotate(-90deg);
+      }
```

Comment:
The values of the `::before` should be the same as the `::after`.
So think you can rewrite it as follows: 

```scss
&::after,
&::before {
  bottom: 0;
  left: 50%;
  transform: translate(-50%, 100%) rotate(-90deg);
}
```